pkg/informer/informer.go

```diff
@@ -219,15 +249,115 @@ func (d *scopedDynamicDiscoverySharedInformerFactory) ForResource(gvr schema.Gro
 	return clusterInformer.Cluster(d.cluster), nil
 }
 
+// NewSingleClusterDynamicDiscoverySharedInformerFactory returns a factory for
```

Comment:
```suggestion
// NewScopedDynamicDiscoverySharedInformerFactory returns a factory for
```

this is the wording used today

Reply:
Done